h2o-docs/src/product/data-science/common.rst

```diff
@@ -1,11 +1,42 @@
-Commonalities
--------------
-
 Quantiles
-~~~~~~~~~
+---------
+
+**Note**: The quantile results in Flow are computed lazily on-demand and cached. It is a fast approximation (max - min / 1024) that is very accurate for most use cases. If the distribution is skewed, the quantile results may not be as accurate as the results obtained using ``h2o.quantile`` in R or ``H2OFrame.quantile`` in Python.
+
+Early Stopping
+--------------
+
+All of the H2O supervised learning algorithms allow for early stopping during model building and scoring. 
+
+Early Stopping in All Supervised Algorithms
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- :ref:`max_runtime_secs` (Defaults to 0/disabled.)
+
+The ``max_runtime_secs`` option specifes the maximum runtime in seconds that you want to allot in order to complete the model. If this maximum runtime is exceeded before the model build is completed, then the model will fail. When performing a grid search, this option specifies the maximum runtime in seconds for the entire grid. This option can also be combined with ``max_runtime_secs`` in the model parameters. If ``max_runtime_secs`` is not set in the model parameters, then each model build is launched with a limit equal to the remainder of the grid time. On the other hand, if ``max_runtime_secs`` is set in the model parameters, then each build is launched with a limit equal to the minimum of the model time limit and the remaining time for the grid.
+
+Early Stopping in AutoML, Deep Learning, DRF, GBM, and XGBoost
```

Comment:
These are also the params for early stopping in a grid search.  Can you add "Grid Search" after "AutoML"?